platform-service-framework/src/main/java/org/hypertrace/core/serviceframework/PlatformService.java

```diff
@@ -133,12 +133,12 @@ public void start() {
     context.addServlet(new ServletHolder(new JVMDiagnosticServlet()), "/diags/*");
 
     final Thread thread = new Thread(this::doStart);
+    thread.setUncaughtExceptionHandler(
```

Comment:
The potential risk here is that there may be some exceptions that _are_ ignorable today and uncaught that would lead to a server quit. IMO we should be fixing any such spots (And kube will recover for us in the interim)